### PR TITLE
[AdminlistBundle] [MediaBundle] Changeablelimitinterface implemented and fixes some bugs

### PR DESCRIPTION
**Pull request by Numkil**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #1900 #1893 

Fixes above tickets and implements changeablelimitinterface in bundles.

### CODE REVIEW

File: src/Kunstmaan/AdminListBundle/Resources/views/AdminListTwigExtension/widget.html.twig [hunk 1]
```diff
@@ -2,8 +2,18 @@
...
 {% include '@KunstmaanAdminList/AdminListTwigExtension/filters.html.twig' %}
 
-{% if adminlist.pagerfanta.haveToPaginate() %}
-    {{ pagerfanta(adminlist.pagerfanta, 'twitter_bootstrap_translated') }}
+{% if adminlist.configurator is instanceof('Kunstmaan\\AdminListBundle\\AdminList\\Configurator\\ChangeableLimitInterface') %}
+<div class="container-fluid">
+    <div class="dropdown pull-right">
+        <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{{ 'kuma_admin_list.limit'|trans({'%limit%': adminlist.configurator.limit}) }}</button>
+        <ul class="dropdown-menu">
+            <li><a href="?limit=10">10</a></li>
```
Comment (Devolicious) [inline src/Kunstmaan/AdminListBundle/Resources/views/AdminListTwigExtension/widget.html.twig]: I would make this configurable

File: src/Kunstmaan/MediaBundle/Resources/views/Folder/show.html.twig [hunk 2]
```diff
@@ -217,6 +216,22 @@
                             </div>
                         </div>
                     </div>
+                    <div class="row list-group">
```
Comment (Devolicious) [inline src/Kunstmaan/MediaBundle/Resources/views/Folder/show.html.twig]: Maybe put this in a general template that is included where needed. That removes duplicate code.

**Comment by Numkil** [comment]
@Devolicious i updated my code according to your suggestions

**Comment by dannyvw** [comment]
@Numkil The target branch is master, so this will not be fixed in version 5.0?

**Comment by Numkil** [comment]
i will make a seperate PR for the bugfix only on 5.0 branch and move this to master branch,

**Comment by dannyvw** [comment]
@Devolicious The linked issues are bugs. I guess the interface is the problem? If so, then that should be submitted to master and a fix should be added to 5.0.

**Comment by Devolicious** [comment]
Agreed

**Comment by Numkil** [comment]
I moved the PR to master branch again and made a new PR in #1927

**Comment by acrobat** [comment]
This PR breaks the pages tab, you get an error directly when opening the pages tab. The `instanceof` function does not exist in twig (unless created yourself)

Error:
> Unknown "instanceof" test.

Use here: https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/f5652ae5b4dda611398b318444c6922b6089a7b9/src/Kunstmaan/AdminListBundle/Resources/views/AdminListTwigExtension/limit.html.twig#L1

**Comment by Numkil** [comment]
It seems like i forgot to commit the twigextension that i created that contained the instanceof method. Im sorry Ill make a pr to add it

**Pull request merged**